src/Microsoft.VisualStudio.AppDesigner/ApplicationDesigner/ApplicationDesignerEditorFactory.vb

```diff
@@ -229,17 +228,9 @@ Namespace Microsoft.VisualStudio.Editors.ApplicationDesigner
             Return hr
         End Function
 
-        Private Function UseNewEditor(vsHierarchy As IVsHierarchy) As Boolean
-            If Not vsHierarchy.IsCapabilityMatch("CPS & CSharp") Then
-                ' The new editor is only available for CPS-based C# projects
-                Return False
-            End If
-
-            Dim featureFlags = TryCast(_siteProvider.GetService(GetType(SVsFeatureFlags)), IVsFeatureFlags)
-
-            Return _
-                featureFlags IsNot Nothing AndAlso _
-                featureFlags.IsFeatureEnabled("CPS.UpdatedProjectPropertiesDesigner.Local", False)
+        Private Shared Function UseNewEditor(vsHierarchy As IVsHierarchy) As Boolean
+            ' The new editor is only available for CPS-based C# projects
+            Return vsHierarchy.IsCapabilityMatch("CPS & CSharp")
```

Comment:
I wonder if we should check for a capability explicitly related to property pages, and then we can "light up" the property pages in other languages/project types just by adding that capability when it is ready. E.g. when the VB property page changes are ready VB projects add that capability.

Reply:
I like this idea, thanks. I've added a new `ProjectPropertiesEditor` capability, and opted C# projects in by default.